commonknowledge/mapped
# Draft: Feature/map 293 editable google sheet readwrite integration

## Description
Adds a new External Data Source: EditableGoogleSheetsSource

## Motivation and Context
https://linear.app/commonknowledge/issue/MAP-293/editable-google-sheet-readwrite-integration

## How Can It Be Tested?
Create a new Google Sheet with some headers: "name", "email", "postcode", "constituency". Fill in some values.
Add it as a data source in mapped. Import it and also update the constituency field. Try adding a new row or changing some data to check the webhooks.

## How Will This Be Deployed?
Normal deployment
